build/build-release.sh

```diff
@@ -0,0 +1,233 @@
+  for jar in $(ls "$PROJECT_DIR/master/target/scala-$SCALA_VERSION/jars"); do
+    (
+      cd $DIST_DIR/master-jars
+      ln -snf "../jars/$jar" .
+    )
+  done
+  ## Copy worker jars
+  cp "$PROJECT_DIR"/worker/target/celeborn-worker_$SCALA_VERSION-$VERSION.jar "$DIST_DIR/worker-jars/"
+  cp "$PROJECT_DIR"/worker/target/scala-$SCALA_VERSION/jars/*.jar "$DIST_DIR/jars/"
+  for jar in $(ls "$PROJECT_DIR/worker/target/scala-$SCALA_VERSION/jars"); do
+    (
+      cd $DIST_DIR/worker-jars
+      ln -snf "../jars/$jar" .
+    )
+  done
+  ## Copy spark client jars
+  cp "$PROJECT_DIR"/client-spark/spark-$SPARK_MAJOR_VERSION-shaded/target/celeborn-client-spark-${SPARK_MAJOR_VERSION}-shaded_$SCALA_VERSION-$VERSION.jar "$DIST_DIR/spark3/"
+  #build 2.4
+
+  VERSION=$("$MVN" help:evaluate -Dexpression=project.version -Pspark-2.4 2>/dev/null | grep -v "INFO" |
+    grep -v "WARNING" |
+    tail -n 1)
+  SCALA_VERSION=$("$MVN" help:evaluate -Dexpression=scala.binary.version -Pspark-2.4 2>/dev/null | grep -v "INFO" |
+    grep -v "WARNING" |
+    tail -n 1)
+  SPARK_VERSION=$("$MVN" help:evaluate -Dexpression=spark.version -Pspark-2.4 2>/dev/null | grep -v "INFO" |
+    grep -v "WARNING" |
+    tail -n 1)
+  SPARK_MAJOR_VERSION=${SPARK_VERSION%%.*}
+  "$MVN" clean package -DskipTests -Pspark-2.4
+  cp "$PROJECT_DIR"/client-spark/spark-$SPARK_MAJOR_VERSION-shaded/target/celeborn-client-spark-${SPARK_MAJOR_VERSION}-shaded_$SCALA_VERSION-$VERSION.jar "$DIST_DIR/spark2/"
+
+  # Copy other things
+  mkdir "$DIST_DIR/conf"
+  cp "$PROJECT_DIR"/conf/*.template "$DIST_DIR/conf"
+  cp -r "$PROJECT_DIR/bin" "$DIST_DIR"
+  cp -r "$PROJECT_DIR/sbin" "$DIST_DIR"
+  mkdir "$DIST_DIR/docker"
+  cp "$PROJECT_DIR/docker/Dockerfile" "$DIST_DIR/docker"
+  cp -r "$PROJECT_DIR/docker/helm" "$DIST_DIR/docker"
+  # Copy notice ,license and disclaimer
+  cp "$PROJECT_DIR/DISCLAIMER" "$DIST_DIR/"
+  cp "$PROJECT_DIR/LICENSE-binary" "$DIST_DIR/LICENSE"
+  cp "$PROJECT_DIR/NOTICE" "$DIST_DIR/NOTICE"
+  ${PROJECT_DIR}/build/collect-licenses.sh $DIST_DIR/jars $DIST_DIR/
+
+  TARDIR_NAME="apache-celeborn-$VERSION-$NAME"
+  TARDIR="$PROJECT_DIR/$TARDIR_NAME"
+  rm -rf "$TARDIR"
+  cp -R "$DIST_DIR" "$TARDIR"
+  tar czf "apache-celeborn-$VERSION-$NAME.tgz" -C "$PROJECT_DIR" "$TARDIR_NAME"
+  rm -rf "$TARDIR"
+
+  if [ "$SKIP_GPG" == "false" ]; then
+    gpg --armor --detach-sig "apache-celeborn-$VERSION-$NAME.tgz"
+  fi
+  $SHASUM "apache-celeborn-$VERSION-$NAME.tgz" >"apache-celeborn-$VERSION-$NAME.tgz.sha512"
+
+}
+
+package_source() {
+
+  VERSION=$("$MVN" help:evaluate -Dexpression=project.version -Pspark-3.3 2>/dev/null | grep -v "INFO" |
+    grep -v "WARNING" |
+    tail -n 1)
+
+  SRC_TGZ_FILE="apache-celeborn-${VERSION}-incubating-source.tgz"
+  SRC_TGZ="${SRC_TGZ_FILE}"
+
+  rm -f "${SRC_TGZ}*"
+
+  echo "Creating source release tarball ${SRC_TGZ_FILE}"
+
+  git archive --prefix="apache-celeborn-${VERSION}-incubating-source/" -o "${SRC_TGZ}" HEAD
```

Comment:
`.gitattributes` is required to filter out those files that should not be included in the release tarball